metecho/api/models.py

```diff
@@ -536,11 +541,12 @@ class Task(
     assigned_qa = JSONField(null=True, blank=True)
 
     slug_class = TaskSlug
+    tracker = FieldTracker(fields=["name"])
 
     def __str__(self):
         return self.name
 
-    def save(self, *args, force_project_save=True, **kwargs):
+    def save(self, *args, force_project_save=False, **kwargs):
```

Comment:
@wlonk Related to the above changes in `should_update_status`... I think this was just a typo before?